### DISCUSSION TITLE
Have UI/Design View for Durable Functions Workflow

### DISCUSSION THREAD

**Discussion by brandonh-msft** [Azure/azure-functions-durable-extension#1776] [Ideas]
# Customer Request

Create a simple UI/Design View for users to use for Durable Functions when going throug the workflow (like that of Logic App Design View).

It enhances the user experience and helps developers debug their Durable Functions more easily.

**Problem Statement**: The partner would like for Durable Functions to have a simple UI or Design View, similar to the Logic App Design View.

**Desired Experience**: Durable Functions should have a Design View similar to the one available for Logic Apps.

**Comment by cgillum** [comment]
The key value-prop of Durable Functions is that you can write workflows in code (i.e. the IDE is your designer). This is why we've stayed away from looking into a UI/design surface so far.

This doesn't mean we're completely against this idea. However, my question for you; what is it that the customer needs which cannot be solved using (as you mentioned) Azure Logic Apps? I'd like to understand that better to know if it makes sense for us to go this route vs. improving the Logic Apps experience.

**Comment by brandonh-msft** [comment]
I think [the Logic Apps Preview](https://docs.microsoft.com/en-us/azure/logic-apps/logic-apps-overview-preview#:~:text=The%20Azure%20Logic%20Apps%20Preview%20runtime%20uses%20Azure,logic%20app%20type%20anywhere%20that%20Azure%20Functions%20runs.) gets us very close to this customer ask today. As this request was put in by a customer before that Preview was publicly available, we can probably close it in favor of that now, especially as the offering evolves (ability to breakpoint & debug an action, etc.).
